### PR TITLE
Imos 2.7.0

### PR DESCRIPTION
**Pull request by julian1**

### CODE REVIEW

**Comment by jonescc** [comment]
Looking at this as well

File: README.md [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+* Netcdf output extension
 
-To build:
-
+### To build:
 ```
 git submodule update --init
-mvn clean install -U -Dmaven.test.skip=true 
+mvn clean
+mvn -P wps,xslt install -DskipTests
```
Comment (jonescc) [inline README.md]: No real reason to use install here - package will build all the artifacts.  We don't use local repositories for artifacts we build.

I'd remove -Dskiptests - we don't want to encourage building without testing.  Thats something the developer can choose to do if they want to speed up building while developing if they know what they are doing.

**Comment by julian1** [comment]
fixed

**Comment by jkburges** [comment]
Don't think this is relevant anymore.

**Pull request closed without merging**